lib/flex.js

```diff
@@ -1,6 +1,7 @@
 const Configuration = require('./configuration/configuration')
 const extractValidationStats = require('./stats/extractValidationStats')
 const Checks = require('./actions/checks')
+const { registerValidatorsAndActions } = require('./registry')
```

Comment:
I personally would much prefer doing a require without the scoping. i.e. 

`const registry = require('.registry')`

it is much more obvious that `registry.registerValidatorsAndActions` is part of another module/object then reading `registerValidatorsAndActions` that can be either a function within this same file or coming from somewhere else.

minor comment and personal preference.

Reply:
done.